genestack_client/files_util.py

```diff
@@ -380,6 +381,48 @@ def share_files(self, accessions, group, destination_folder=None, password=None)
         if destination_folder is not None:
             share_utils.invoke('linkFiles', accessions, destination_folder, group)
 
+    def share_folder(self, folder_accession, group, destination_folder=None, password=None):
+        """
+        Recursively share folder.
+
+        This method make repeated calls to server, each method call
+        shares chunk of the files in the folder that aren't shared yet with ``group` or ``WORLD``.
+
+        Due to the indexing lag, same files can be included in different calls.
+        Method implementation is written to overcome this limitation.
+        Sum of the counts returned by all call is greater of equal than the count of files shared.
+
+        Method can work for a long time if ``folder`` contains a lot of files.
```

Comment:
what's this for?

Reply:
It can work for couple of hours without any output.